pom.xml

```diff
@@ -215,9 +215,9 @@
     </profile>
 
     <profile>
-      <id>java12</id>
+      <id>java9+</id>
       <activation>
-        <jdk>[12,)</jdk>
+        <jdk>[9,)</jdk>
```

Comment:
IMHO it can be even in parent pom

Reply:
> IMHO it can be even in parent pom

Agree. Once it is - this should be dropped.